zulipterminal/ui_tools/views.py

```diff
@@ -293,13 +293,17 @@ def __init__(self) -> None:
 class StreamsView(urwid.Frame):
     def __init__(self, streams_btn_list: List[Any], view: Any) -> None:
         self.view = view
-        self.log = urwid.SimpleFocusListWalker(streams_btn_list)
+        # Create Stream List Box
         self.streams_btn_list = streams_btn_list
-        self.focus_index_before_search = 0
+        self.log = urwid.SimpleFocusListWalker(streams_btn_list)
         list_box = urwid.ListBox(self.log)
+        # Create Stream Search Box
         self.stream_search_box = PanelSearchBox(self,
                                                 'SEARCH_STREAMS',
                                                 self.update_streams)
+        self.focus_index_before_search = 0
```

Comment:
A better default value would be `None`, I gather. That would indicate that we didn't have any searches yet.

Reply:
This would cause an error with `self.log.set_focus()` as it accepts only `int`. ZT can crash in that case

Reply:
Right, that would require extra logic. The current implementation looks good.